planning/behavior_velocity_blind_spot_module/src/manager.cpp

```diff
@@ -72,8 +72,6 @@ void BlindSpotModuleManager::launchNewModules(
       module_id, lane_id, planner_data_, planner_param_, logger_.get_child("blind_spot_module"),
       clock_));
     generateUUID(module_id);
-    updateRTCStatus(
-      getUUID(module_id), true, std::numeric_limits<double>::lowest(), path.header.stamp);
```

Comment:
If this line is removed, RTC status does not contain this information.
To check the existence of a module with `module_id`, status should contain this information, so I think this information including distance as `-10000.0` is required.